tests/test_api.py

```diff
@@ -858,6 +858,38 @@ def test_boolean_filters_on_boolean_field(self):
 
                 )
 
+    def test_sort_relation_field(self):
+        url = '/users/?sort[]=location.name'
+        with self.assertNumQueries(1):
+            response = self.client.get(url)
+        self.assertEquals(200, response.status_code)
+        self.assertEquals(
+            {
+                'users': [
+                    {'id': 1, 'location': 1, 'name': '0'},
+                    {'id': 2, 'location': 1, 'name': '1'},
+                    {'id': 3, 'location': 2, 'name': '2'},
+                    {'id': 4, 'location': 3, 'name': '3'},
+                ]
+            },
+            json.loads(response.content.decode('utf-8')))
+
+    def test_sort_relation_field_reverse(self):
+        url = '/users/?sort[]=-location.name'
+        with self.assertNumQueries(1):
+            response = self.client.get(url)
+        self.assertEquals(200, response.status_code)
+        self.assertEquals(
+            {
+                'users': [
+                    {'id': 4, 'location': 3, 'name': '3'},
+                    {'id': 3, 'location': 2, 'name': '2'},
+                    {'id': 1, 'location': 1, 'name': '0'},
+                    {'id': 2, 'location': 1, 'name': '1'},
```

Comment:
Not sure how stable the last 2 items are and the potential for intermittent test failures. Maybe we can do something like:

```python
response = json.loads(response.content.decode('utf-8')))
self.assertEquals(
    [3, 2, 1, 1],
    [row['location'] for row in response['users']]
)
```

Reply:
Ah. Good call. Fixed!